src/pykeen/utils.py

```diff
@@ -990,6 +990,36 @@ def get_expected_norm(
         raise TypeError(f"norm not implemented for {type(p)}: {p}")
 
 
+def artanh(x):
```

Comment:
is this the same as [`torch.arctanh`](https://pytorch.org/docs/stable/generated/torch.arctanh.html)?

Reply:
Should be: https://mathworld.wolfram.com/InverseHyperbolicTangent.html

Reply:
yes, cf. [here](https://en.wikipedia.org/wiki/Inverse_hyperbolic_functions#Inverse_hyperbolic_tangent)